app/tests/unit/controllers/roadmap.spec.ts

```diff
@@ -22,7 +22,7 @@ afterEach(() => {
   jest.resetAllMocks();
 });
 
-const CURRENT_CONTEXT = { authType: 'BEARER', tokenPayload: null, userId: 'abc-123' };
+const CURRENT_CONTEXT = { authType: 'BEARER', bearerToken: 'sometoken', tokenPayload: null, userId: 'abc-123' };
```

Comment:
Test coverage: Consider testing scenarios where the `bearerToken` has a falsy value (null and/or undefined) and making sure that your code is still behaving as anticipated.